internal/reconciliations/istio_resources/gateway_kyma.yaml

```diff
@@ -0,0 +1,31 @@
+apiVersion: networking.istio.io/v1alpha3
+kind: Gateway
+metadata:
+  name: kyma-gateway
+  namespace: kyma-system
+spec:
+  selector:
+    app: istio-ingressgateway
+    istio: ingressgateway # use istio default ingress gateway
+  servers:
+  - port:
+      number: 443
+      name: https
+      protocol: HTTPS
+    tls:
+      mode: SIMPLE
+      credentialName: kyma-gateway-certs
+      cipherSuites:
+      - ECDHE-RSA-CHACHA20-POLY1305
+      - ECDHE-RSA-AES256-GCM-SHA384
+      - ECDHE-RSA-AES128-GCM-SHA256
```

Comment:
We should be able to remove those
```suggestion
```

Reply:
Yep, those are default see: https://github.com/orgs/kyma-project/projects/27/views/1?pane=issue&itemId=30035511